src/app/shared/page/page.component.html

```diff
@@ -11,17 +11,8 @@ <h1 class="text-center text-light text-uppercase m-0" style="line-height:15rem;"
     {{ loadContent() }}
   </div>
 
-  <div class="container mt-5" style=""> <!-- AUTHOR SECTION -->
-    <div class="row pl-md-5 pt-3 pb-3" style="">
-      <div class="col">
-        <div class="float-left h-100"><div class="mr-3 d-inline-block" style="height:50px;width:50px;border-radius:50%;overflow:hidden;"><img class="" style="width:50px;height:auto;" src="{{ MEDIA + '/' + owner.photo }}"></div></div>
-        <div class="d-inline-block align-middle">
-          <h6 class="m-0"><a href="{{ MASK }}/{{page.owner.username}}">{{ owner.full_name }}</a></h6>
-          <a class="m-0" href="mailto:{{ owner.email }}">{{ owner.email }}</a>
-        </div>
-        
-      </div>
-    </div>
+  <div id="author" class="container mb-3 mt-5 pl-md-5"> <!-- AUTHOR SECTION -->
+    <profile-info *ngIf="pageProfile" [profile]="pageProfile" [showPhoto]="true" [showEmail]="true" [fields]="['majors', 'minors']"></profile-info>
```

Comment:
This line is throwing an error. But I think the root cause might be with the way that we managing inserting content html. I've been experimenting a little bit with how our bypassSecurity pipe works and I think that it might be better to go that route.